apps/demo/src/app/checkout/components/place-order/place-order.component.spec.ts

```diff
@@ -11,7 +11,8 @@ import { DaffDriverTestingModule } from '@daffodil/driver/testing';
 import * as fromDemoCheckout from '../../reducers';
 import { PlaceOrderComponent } from './place-order.component';
 
-describe('PlaceOrderComponent', () => {
+// Because the fromCart selector is now being subscribed to, any test throws an error. Skipping tests until fromCart can be mocked.
```

Comment:
Would we be able to import the `cart` state here and be able to test this? I'm aware that's increases the surface area more than necessary, but its better than just `xdescribing` the whole test suite. I've opened #294 to address this issue more thoroughly later.

Reply:
ok don't laugh. Turns out the tests work if I add the cart store to the StoreModule for the spec even if I import from `@daffodil/cart`. I just can't spy on the selectors or the tests will break.